.codesandbox/ci.json

```diff
@@ -3,5 +3,6 @@
     "/examples/create-react-app",
     "/examples/create-react-app-with-typescript"
   ],
-  "silent": true
+  "silent": true,
+  "node": "16"
```

Comment:
Is Node 16 a requirement? if so, should we update Elyra's readme that currently mentions Node 12+?

Reply:
IIRC, there was an issue in CI that caused us to need to set a version. I'm unsure if it needs to be 16 or not though.

Reply:
This is just for code sandbox so it's not a new requirement for elyra.